### DISCUSSION TITLE
100kB ping body limit

### DISCUSSION THREAD

**Discussion by Sdub76** [healthchecks/healthchecks#650] [Q&A]
I still don't see the 100kB body limit increase in the self-hosted version of healthchecks.  Is there usually a delay in features making it into this version of the product?  Or is there a setting I need to change to enable it?

**Comment by cuu508** [comment]
The ping body limit is a configuration parameter. The default is still 10KB but you can set it to a higher value.

See https://healthchecks.io/docs/self_hosted_configuration/#PING_BODY_LIMIT
